.teamcity/src/main/kotlin/util/AdHocPerformanceScenario.kt

```diff
@@ -48,6 +48,8 @@ abstract class AdHocPerformanceScenario(os: Os) : BuildType({
             allowEmpty = false,
             description = "Which performance test to run. Should be the fully qualified class name dot (unrolled) method name. E.g. org.gradle.performance.regression.java.JavaUpToDatePerformanceTest.up-to-date assemble (parallel true)"
         )
+        text("testJavaVersion", "8",  display = ParameterDisplay.PROMPT, allowEmpty = false, description = "The java version to run the performance tests, e.g. 8/11/17")
+        text("testJavaVendor", "oracle",  display = ParameterDisplay.PROMPT, allowEmpty = false, description = "The java vendor to run the performance tests, e.g. oracle/openjdk/adoptium")
```

Comment:
Can you make the vendor a dropdown as in the Rerun Flaky test configuration? https://github.com/gradle/gradle/blob/c283a8b5504c0db9e02696ad61df98177e45ea26/.teamcity/src/main/kotlin/util/RerunFlakyTest.kt#L93-L99